src/main/resources/logback.xml

```diff
@@ -9,7 +9,7 @@
             class="ch.qos.logback.core.ConsoleAppender">
     <layout class="ch.qos.logback.classic.PatternLayout">
       <Pattern>
-        %black(%d{ISO8601}) %highlight(%-5level) [%blue(%t)] %yellow(%C{1.}): %msg%n%throwable
+        %black(%d{ISO8601}) %highlight(%-5level) [%blue(%t)] %yellow(%C{1}): %msg%n%throwable
```

Comment:
Was getting a `NumberFormatException` for `1.`, during the logback startup. This was the only place where I found the `1.`. Changing it the build didn't fail in the same place, but will have to confirm what was the 1., and if the 1 alone will act the same way.